teku/src/main/java/tech/pegasys/teku/cli/subcommand/ValidatorClientCommand.java

```diff
@@ -158,6 +158,6 @@ public TekuConfiguration tekuConfiguration() {
     loggingOptions.configureWireLogs(builder);
     interopOptions.configure(builder);
     metricsOptions.configure(builder);
-    return builder.build();
+    return builder.build(false);
```

Comment:
I'm not a huge fan of passing in context into build() functions. I'd prefer that we have parameters on the builder that we can set, then call build with no parameters, which is a much more 'normal' builder pattern...

Reply:
ye I agree, another option could be to have two different `build`, `buildForBeaconNode` `buildForValidatorClient` and drop the simple `build`

Reply:
My point on this was that options are the same for bn and vc but we don't need the same config for vc and most subcommands as we need for bn, but I'm not 100% on this, yeah, if it'll branch in future, this approach is definitely bad, I will check how this could look like in atomic kzg disabling for this case